schemas/com.snowplowanalytics.snowplow.ecommerce/product/jsonschema/1-0-0

```diff
@@ -0,0 +1,105 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for a product entity in Ecommerce",
+    "self": {
+        "vendor": "com.snowplowanalytics.snowplow.ecommerce",
+        "name": "product",
+        "format": "jsonschema",
+        "version": "1-0-0"
+    },
+    "type": "object",
+    "properties": {
+        "id": {
+            "description" : "The SKU or product ID.",
+            "type": "string",
+            "maxLength": 4096
+        },
+        "name" : {
+            "description" : "The name or title of the product.",
+            "type": ["string", "null"],
+            "maxLength": 4096
+        },
+        "category": {
+            "description" : "The category the product belongs to. Use a consistent separator to express multiple levels. E.g. Woman/Shoes/Sneakers",
+            "type": "string",
+            "maxLength": 4096
+        },
+        "price": {
+            "description" : "The price of the product at the current time.",
+            "type": "number",
+            "minimum": 0,
+            "multipleOf": 0.01,
+            "maximum": 9999999
+        },
+        "list_price": {
+            "description" : "The list or recommended retail price of a product.",
+            "type" : ["number", "null"],
+            "minimum": 0,
+            "multipleOf": 0.01,
+            "maximum": 9999999
+        },
+        "quantity" : {
```

Comment:
Currently, this will get stored in the warehouse as a 64-bit integer.  If you put a `maximum` on it, then you can make it get stored as a 32-bit integer -- if that's something you would prefer.